### ISSUE TITLE
Bug with parent word inside a namespace

### ISSUE THREAD

**Issue by tristanbes** [FriendsOfPHP/PHP-CS-Fixer#3935]
#### The PHP version you are using (`$ php -v`):
`7.2.7`

#### PHP CS Fixer version you are using (`$ php-cs-fixer -V`):
`v2.12.2`

#### The command you use to run PHP CS Fixer:
`php-cs-fixer fix`
#### The configuration file you are using, if any:
```php
<?php

$finder = PhpCsFixer\Finder::create()
    ->in([__DIR__.'/src'])
    ->exclude(['Tests'])
;

return PhpCsFixer\Config::create()
    ->setRiskyAllowed(true)
    ->setRules([
        '@Symfony' => true,
        'no_unused_imports' => false,
        'array_syntax' => [
            'syntax' => 'short',
        ],
        'binary_operator_spaces' => [
            'align_double_arrow' => true,
            'align_equals' => true
        ],
        'no_unreachable_default_argument_value' => false,
        'braces' => [
            'allow_single_line_closure' => true,
        ],
        'heredoc_to_nowdoc' => false,
        'phpdoc_summary' => false,
        'yoda_style' => false,
        'native_constant_invocation' => true,
        'native_function_invocation' => [
            'include' => ['@compiler_optimized'],
            'scope'   => 'namespaced',
        ],
        'no_superfluous_phpdoc_tags' => true,
    ])
    ->setFinder($finder)
;

```

#### If applicable, please provide minimum samples of PHP code (as plain text, not screenshots):
 * before running PHP CS Fixer (no changes):
```php
namespace Yprox\Model\Site\Parent;
```

 * with unexpected changes applied when running PHP CS Fixer:
```php
namespace Yprox\Model\Site\parent;
```

 * with the changes you expected instead:
```php
no changes
```

**Comment by keradus** [comment]
using semi-reserved words as namespace is not best idea imho :(

**Comment by kmvan** [comment]
Why not rename the Parent namespace?

**Comment by kubawerlos** [comment]
No matter if it's a good idea or bad it is a bug -> https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3937

**Comment by ntzm** [comment]
Can be closed now, is fixed as of 2.12.3
